src/components/search/omnibox-directions.component.tsx

```diff
@@ -38,16 +44,16 @@ import UtilityService from "../../services/utility.service";
  */
 export interface OmniboxDirectionsProps {
   currentLocation: Location;
-  startLocation: MarkerLocation;
-  endLocation: POI;
-  setStartLocation: (marker: MarkerLocation) => void;
-  setEndLocation: (poi: POI) => void;
+  startLocation: MarkerLocation | Building;
+  endLocation: POI | Building;
```

Comment:
Same as before, endLocation should be of type `MarkerLocation | Building`.

Reply:
Same as above, in commit 8804195